spec/models/content_item_spec.rb

```diff
@@ -263,6 +263,35 @@
         expect(@item.errors[:routes]).to eq(["redirect items cannot have routes"])
       end
     end
+
+    describe 'phase' do
+
+      context 'a content item without a phase' do
+        let(:content_item) { build(:content_item) }
+
+        it 'is valid' do
+          expect(content_item).to be_valid
+        end
+      end
+
+      context 'a content item with an alpha or beta phase' do
+        let(:alpha_content_item) { build(:content_item, phase: 'alpha') }
+        let(:beta_content_item) { build(:content_item, phase: 'beta') }
+
+        it 'is valid' do
+          expect(alpha_content_item).to be_valid
+          expect(beta_content_item).to be_valid
+        end
+      end
+
+      context 'a content item with a phase other than alpha or beta' do
+        let(:content_item) { build(:content_item, phase: 'not-a-correct-phase') }
+
+        it 'is invalid' do
+          expect(content_item).to_not be_valid
+        end
+      end
+    end
```

Comment:
Sorry to nitpick, but this would probably be best up with the other "regular" fields (maybe after the `content_id` validations).  It would also be good if it followed the pattern of the other validations - it makes it easier to read if all the tests in a given file follow a similar pattern.